main/src/main/res/values/strings.xml

```diff
@@ -921,6 +921,8 @@
     <string name="settings_ec_description">Authorize c:geo with extremcaching.com to search for caches and access/filter your found caches.</string>
     <string name="settings_su_description">Authorize c:geo with geocaching.su to search for caches.</string>
     <string name="settings_lc_description">Include Adventure Lab caches on the map.\n\n(Available to \"Geocaching.com\" premium members only. Requires an activated \"Geocaching.com\" service.)\n\nChanging this setting requires a restart.</string>
+    <string name="settings_lc_found_state_title">Found State Handling</string>
```

Comment:
```suggestion
    <string name="settings_lc_found_state_title">Manual found state</string>
```

Our preference names are typically describing the "if enabled"-functionality and only the first word is capitalized.

@baiti Despite from this, is the PR ready to merge?

Reply:
(BTW, just noticed: pretty cool reviewing feature to make code suggestions which can directly be committed with a single click via github.com UI)

Reply:
I appreciate your suggestions and will integrate those shortly, however, taking the classic approach with `git commit`, `git push`.

Wrt readiness: I'd consider it ready but let's wait till @murggel had a chance to do a test.

Reply:
Unfortunately I can't test it right now, because I am waiting for @lineflyer to get the consumer key - probably he has overseen my request...
I don't know, if some one of you can give it to me.

Reply:
@murggel 
have a look at your inbox

Reply:
You could have done a `git pull` with the PR branch checked out, or use the equivalent button in your IDE if you prefer (sometimes also called "update project" instead of "pull")

Reply:
I am currently testing ... will give feedback probably tomorrow evening